consensus/statehash.go

```diff
@@ -18,9 +18,9 @@ func (s *State) sortedUtxoSet() (sortedOutputs []Output) {
 	for _, utxoString := range unspentOutputStrings {
 		var outputID OutputID
 		copy(outputID[:], utxoString)
-		output, err := s.output(outputID)
-		if err != nil {
-			panic(err)
+		output, exists := s.output(outputID)
+		if !exists {
+			panic("output doesn't exist?")
```

Comment:
go up two function calls?

It's only ever called during testing and when hashing the state. Right now, the state is only ever hashed in debug mode. We might fix this by adding a new rule to consensus about including the state hash in blocks.